api/src/main/java/com/codeforcommunity/api/IProcessor.java

```diff
@@ -11,6 +12,16 @@
 
 public interface IProcessor {
 
+  List<NewsReturn> getAllNews();
+
+  boolean createNews(String title, String description, String author, LocalDateTime date, String content);
```

Comment:
This is where you should create a DTO object that encapsulates all this information so that you can only have one param for these methods rather than 5.